index.html

```diff
@@ -18,246 +18,311 @@
+              <li>
+                <a
+                  href="https://www.linkedin.com/in/belal-gamal-79b8a2133/"
+                  target="_blank"
+                  rel="noopener"
+                >
+                  <img src="./images/linkedin.svg" alt="linkedin-icon" />
+                </a>
+              </li>
+              <li>
+                <a href="#" target="_blank" rel="noopener">
+                  <img src="./images/Vector.svg" alt="vector-icon" />
+                </a>
+              </li>
+              <li>
+                <a
+                  href="https://github.com/belalgamal46"
+                  target="_blank"
+                  rel="noopener"
+                >
+                  <img src="./images/github.svg" alt="github-icon" />
+                </a>
+              </li>
+              <li>
+                <a href="#" target="_blank">
+                  <img src="./images/handVector.svg" alt="hand-vector-icon" />
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
       </section>
 
       <section class="works">
-        <section class="card-works">
-          <div class="snapshoot-portfolio">
-            <img src="./images/tonic.jpg" alt="tonic-image" />
-          </div>
-          <div class="portofolio-details">
-            <div class="primary-text">
-              <h2>Tonic</h2>
-              <div class="frame">
-                <ul>
-                  <li>CANOPY</li>
-                  <li>Back End Dev</li>
-                  <li>2015</li>
+        <div class="cards-container">
+          <article class="card-works">
+            <div class="snapshoot-portfolio">
+              <img
+                src="./images/tonic.jpg"
+                alt="tonic-image"
+                class="mobile-image"
+              />
+              <img
+                src="./images/Desktop-tonic.jpg"
+                alt="multi-post-stories-image"
+                class="desktop-image"
+              />
+            </div>
+            <div class="portofolio-details">
+              <div class="primary-text">
+                <h2>Tonic</h2>
+                <div class="frame">
+                  <ul>
+                    <li>CANOPY</li>
+                    <img src="./images/Counter.svg" alt="separator" />
+                    <li>Back End Dev</li>
+                    <img src="./images/Counter.svg" alt="separator" />
+                    <li>2015</li>
+                  </ul>
+                </div>
+              </div>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <div class="tags">
+                <ul class="tag">
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
                 </ul>
               </div>
+              <button type="button" class="btn">See Project</button>
```

Comment:
- [ ] Kindly make sure also you implement the button interactions like `hover`, `disable`, and `active` as the Figma design